recipes-example/template-library/template-library.bb

```diff
@@ -0,0 +1,25 @@
+#
+#   Copyright (C) 2017 Pelagicore AB
+#
+
+DESCRIPTION = "A template C/C++ library"
+HOMEPAGE = "https://github.com/Pelagicore/template-library"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=815ca599c9df247a0c7f619bab123dad"
+
+PR = "r0"
+PV = "1.0+git${SRCREV}"
+
+SRC_URI = "gitsm://github.com/Pelagicore/template-library.git;protocol=https;branch=master"
```

Comment:
I think you made a mistake using gitsm here because the repo doesn't seem to have any submodules.